Code/Framework/AzToolsFramework/AzToolsFramework/ComponentMode/ComponentModeCollection.cpp

```diff
@@ -216,6 +216,12 @@ namespace AzToolsFramework
             m_componentModeState = ComponentModeState::Active;
             m_adding = false;
 
+            // notify other systems ComponentMode has started
+            EditorComponentModeNotificationBus::Event(
+                GetEntityContextId(),
+                &EditorComponentModeNotifications::ComponentModeStarted,
+                m_activeComponentTypes[m_selectedComponentModeIndex]);
+
```

Comment:
update: This can be removed, see `m_viewportEditorModeTracker->ActivateMode({ GetEntityContextId() }, ViewportEditorMode::Component);` below which is already sending the notification